// Copyright 2015 gRPC authors.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

syntax = "proto3";

//option java_multiple_files = true;
//option java_outer_classname = "RpcFaceRec";

package face_rec_rpc;

// The greeting service definition.
service FaceRecognition{
  // Sends a greeting
  rpc face_rec_str (MyStr) returns (Image) {}
}

// The request message containing the user's name.
message Image{
  int32 id = 1;
  string name = 2;
  string format = 3;
  string stream = 4;
}

message MyStr{
  string name = 1;
}
// The response message containing the greetings

